test/e2e/framework/service/jig.go

```diff
@@ -679,9 +679,9 @@ func (j *TestJig) AddRCAntiAffinity(rc *v1.ReplicationController) {
 }
 
 // CreatePDB returns a PodDisruptionBudget for the given ReplicationController, or returns an error if a PodDisruptionBudget isn't ready
-func (j *TestJig) CreatePDB(rc *v1.ReplicationController) (*policyv1.PodDisruptionBudget, error) {
+func (j *TestJig) CreatePDB(rc *v1.ReplicationController) (*policyv1beta1.PodDisruptionBudget, error) {
```

Comment:
Checking callers, I couldn't find anyone who actually cared about `CreatePDB` responses besides the `error`.  Can we just drop the returned PDB?

Reply:
I don't mind doing that. However it might just be easier to switch the return types when we are rebasing again.

Reply:
if we drop the PDB return, we'd never have to touch this line again ;)

Reply:
Sorry, I was referring to rest of the PDB methods. They need to change the return types too.